tests/conftest.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+"""Pytest fixtures and other things."""
+
+import os
+import shutil
+
+import pytest
+
+from tests import TEST_DOCS_SRC
+
+
+pytest_plugins = 'sphinx.testing.fixtures'
+
+
+@pytest.fixture
+def firefox_options(firefox_options):
+    firefox_options.add_argument('-headless')
+    return firefox_options
+
+
+@pytest.fixture
+def chrome_options(chrome_options):
+    chrome_options.add_argument('-headless')
+    return chrome_options
+
+
+@pytest.fixture(scope='function', autouse=True)
+def remove_build_folder():
+    """Remove _build folder, if exist."""
+    _build_path = os.path.join(TEST_DOCS_SRC, '_build')
+
+    def delete_build_path():
+        if os.path.exists(_build_path):
+            shutil.rmtree(_build_path)
+
+    delete_build_path()
+    yield
+    delete_build_path()
```

Comment:
These few lines could use a comment, it doesn't make total sense to me.

Reply:
I will add comments for that.
Actually the function before yield runs before each test and the function after yield runs after each test case.
So.. I want to delete the `_build` before and after each test case so that each test has its own `_build` folder and does not depend on the `_build` folder generated by some other test case.

Reply:
Comments added.